codereview.java

```diff
@@ -0,0 +1,37 @@
+public class RuntimeIssuesDemo {
+
+    public static void main(String[] args) {
+        // 1. ArrayIndexOutOfBoundsException
+        int[] numbers = {1, 2, 3};
+        System.out.println(numbers[3]); // Accessing index 3, which is out of bounds
+
+        // 2. NullPointerException
+        String str = null;
+        System.out.println(str.length()); // Attempting to invoke method on null reference
```

Comment:
<b>code: System.out.println(str.length()); // Attempting to invoke method on null reference - </b>
Add a null check before invoking methods on 'str'